README.md

```diff
@@ -350,6 +350,10 @@ rabbitmq.config SSL verify setting.
 
 rabbitmq.config `fail_if_no_peer_cert` setting.
 
+####`ssl_versions`
+
+Choose which SSL versions to enable. Example: `['tlsv1.2', 'tlsv1.1']`
```

Comment:
Could you also add a link to the rabbitmq SSL guide (https://www.rabbitmq.com/ssl.html) and note the recommendation to turn off SSLv3 and TLSv1.0?